Symbolic links error

What's the reason for doing this https://github.com/thymikee/linaria-jest/blob/master/package.json#L11?

I think it might be causing an NPM error.

To replicate, install `linaria-jest` and then try to install any other package:
```sh
npm ERR! code ELOOP
npm ERR! errno -62
npm ERR! syscall open
npm ERR! ELOOP: too many symbolic links encountered, open '/example/node_modules/linaria-jest/node_modules/linaria/npm-shrinkwrap.json'
```

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "linaria-jest",
-  "version": "0.0.4",
+  "version": "0.0.3",
   "description": "Jest utilities for Linaria",
   "author": "Michał Pierzchała <thymikee@gmail.com>",
   "repository": "github:thymikee/linaria-jest",

--- a/package.json
+++ b/package.json
@@ -7,8 +7,7 @@
   "license": "MIT",
   "scripts": {
     "test": "jest",
-    "lint": "eslint . --cache",
-    "postinstall": "yarn link linaria"
+    "lint": "eslint . --cache"
   },
   "dependencies": {
     "babel-preset-env": "^1.6.0",

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "linaria-jest",
-  "version": "0.0.3",
+  "version": "0.0.4",
   "description": "Jest utilities for Linaria",
   "author": "Michał Pierzchała <thymikee@gmail.com>",
   "repository": "github:thymikee/linaria-jest",
@@ -11,7 +11,7 @@
   },
   "dependencies": {
     "babel-preset-env": "^1.6.0",
-    "linaria": "^0.1.4",
+    "linaria": "^0.5.0",
     "prettier": "^1.7.4"
   },
   "devDependencies": {

